src/amp-url-creator.js

```diff
@@ -35,6 +46,55 @@ const HAS_RTL_CHARS = new RegExp('[' + RTL_CHARS + ']');
 /** @private {number} */
 const MAX_DOMAIN_LABEL_LENGTH_ = 63;
 
+
+/**
+ * Constructs a Viewer proxy url using these rules:
```

Comment:
The linked doc is called "cache". probably better to use "cache" throughout instead of "proxy".